src/server/tsconfig.json

```diff
@@ -2,16 +2,11 @@
   "compilerOptions": {
       "target": "es6",
       "module": "commonjs",
-      "outDir": "../../dist",
+      "outDir": "../../dist/server",
       "strict": true,
       "esModuleInterop": true,
       "skipLibCheck": true,
-      "forceConsistentCasingInFileNames": true,
-      "paths": {
-          "@shared/*": [
-              "../shared/*"
```

Comment:
Even though I recently took out the authorization, which motivates the shared alias, I'd like to keep this in here. I foresee adding authorization back in after the talk, or maybe some additional server-side behavior.